### DISCUSSION TITLE
I can't use copilot. It shows expired.

### DISCUSSION THREAD

**Discussion by buraksumer** [community/community#128278] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I am experiencing issues similar to the ones shown in the attached image, and Visual Studio Code is giving me errors. When I researched this problem, I found suggestions that it might be blocked, but I couldn't find any concrete information. I have created a support ticket but haven't received a response yet.

Even when I click the purchase button, the page does not work. I am using two computers (a notebook and a desktop).

I am currently using an older version of Copilot because the latest version was giving errors. I reverted to an older version based on recommendations that it would work.

I would appreciate any assistance you can provide in resolving this issue and understanding its cause.

Support ticket nr: 2825624

![image](https://github.com/community/community/assets/5336421/9bc5a5df-6b80-45be-b201-3528dcda4b19)

**Comment by tpope** [comment]
> I am currently using an older version of Copilot because the latest version was giving errors. I reverted to an older version based on recommendations that it would work.

What version of the extension did you revert to? And what's the version number that's showing up as the latest?

The latest version is 1.200.0. If you're not seeing that version, you probably need to update VS Code.

**Comment by buraksumer** [comment]
well, but I think it can't be a coincidence that it suddenly stopped and now I see this warning when I go to the copilot page 

version 1.155.0

**Comment by buraksumer** [comment]
I wrote in the support section, but I still can't get an answer. I can't even buy paid, isn't that a bit strange? I can't figure out who I can get support from. I press all the buttons in the game, like passing a level :)

**Comment by buraksumer** [comment]
I think it's an unresolved issue. Isn't anyone supporting? I can't get ticket nor response.

**Comment by buraksumer** [comment]
It's been 3 weeks. there is no return. I don't understand why I can't get support on this issue. if I am a company and I buy it, is this how you support me?
